src/binomial.jl

```diff
@@ -0,0 +1,103 @@
+function stirlerr(n::Int64)

```

Comment:
This function is written for `Float64` which should probably be indicated somehow such that it is not suddenly called as part of a `BigFloat` computation. Maybe a way to do that would be to use `n::Float64` instead of `n::Int64`.